genai-perf/README.md

```diff
@@ -508,6 +508,7 @@ when `--output-tokens-mean` is provided. (default: `0`)
 The seed used to generate random values. (default: `0`)
 
 ##### `--request-count <int>`
+##### `--num-requests <int>`
```

Comment:
Added aliases. While PA uses request-count, all other GAP args that use counts start with --num (most notably, --num-prompts).

We can decide for GA what arg name to use (if only one), but having this consistency would make it less confusing to be adding args. I often would type out the wrong one, so I imagine some users might too.